src/osd/PGLog.cc

```diff
@@ -793,8 +797,8 @@ void PGLog::_write_log_and_missing(
   eversion_t dirty_to,
   eversion_t dirty_from,
   eversion_t writeout_from,
-  const set<eversion_t> &trimmed,
-  const set<string> &trimmed_dups,
+  set<eversion_t> *trimmed,
```

Comment:
if my argument holds, we can remove these two parameters. and remove the following code you are changing in this PR.

Reply:
this is not addressed.

Reply:
@tchaikov - your arguments are valid for _write_log_and_missing_wo_missing - there is just a single call of it. But for _write_log_and_missing trimmed/trimmed_dups are still applicable and are used when pg log trimming occurs. Hope I haven't missed something...

Reply:
@tchaikov - ping!

Reply:
@ifed01 sorry for the latency, will review it early tomorrow.